tasks/main.yml

```diff
@@ -106,11 +106,32 @@
     group: "{{  odoo_role_odoo_group }}"
   notify: restart odoo
 
+- name: Build the list of new modules to install
+  become: yes
+  become_user: "{{ odoo_role_odoo_user }}"
+  shell: >
+    psql {{ odoo_role_odoo_db_name }} -tAc
+    "SELECT name
+    FROM (VALUES ('{{ odoo_role_odoo_core_modules.split(',') | join(joiner) }}'))
```

Comment:
`join(joiner)` doesn't tell much, can you think about other name? :)

Reply:
it's called `separator` now